extension/test/integration/cancel-or-refund.handler.spec.js

```diff
@@ -1,62 +1,91 @@
 const { expect } = require('chai')
 
-const testUtils = require('../test-utils')
 const iTSetUp = require('./integration-test-set-up')
 const ctpClientBuilder = require('../../src/ctp/ctp-client')
+const {
+  CTP_ADYEN_INTEGRATION, CTP_PAYMENT_CUSTOM_TYPE_KEY, CTP_INTERACTION_TYPE_CANCEL_OR_REFUND
+} = require('../../src/config/constants')
 
-describe.skip('Cancel or refund', () => {
+describe('::cancelOrRefund::', () => {
   let ctpClient
 
+  let payment
+
   beforeEach(async () => {
     ctpClient = ctpClientBuilder.get()
     await iTSetUp.initServerAndExtension(ctpClient)
+
+    const makePaymentRequestDraft = {
+      amount: {
+        currency: 'EUR',
+        value: 1000
+      },
+      reference: 'YOUR_ORDER_NUMBER',
+      paymentMethod: {
+        type: 'scheme',
+        encryptedCardNumber: 'test_4111111111111111',
+        encryptedExpiryMonth: 'test_03',
+        encryptedExpiryYear: 'test_2030',
+        encryptedSecurityCode: 'test_737'
+      },
+      returnUrl: 'https://your-company.com/'
+    }
+    const paymentDraft = {
```

Comment:
Is there an `authorizationTransaction` in this payment? I saw that you later use this transaction to get `originalReference` and according to the docs, it's a must have param.

Reply:
No, it will be created by integration, 2 phases. So this is before cancel or refund, that way integration test will make a payment first _**before**_ the test and then payment could have an originalReference(pspReference).